modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -59,7 +60,24 @@ $ oc -n openshift-nmstate delete nmstate nmstate
 $ oc delete --all deployments --namespace=openshift-nmstate
 ----
 
-. Delete all the custom resource definition (CRD), such as `nmstates`, that exist in the `nmstate.io` namespace by running the following commands:
+. After you deleted the `nmstate` CR, remove the `nmstate-console-plugin` console plugin name from the `console.operator.openshift.io/cluster` CR.
++
+.. Use the `jq` CLI tool to store the position of the `nmstate-console-plugin` entry that exists among the list of enable plugins. The following command stores the index of the entry in an environment variable named `INDEX`:
```

Comment:
Could you rewrite the first sentence to mention explicit instruction to run a command? I feel like you could remove mention of `jq` since the context of the new prereq (and the use of jq in the command) is sufficient. Up to you, but I feel something like this might be more in line with style guides:
```suggestion
.. Store the position of the `nmstate-console-plugin` entry that exists among the list of enable plugins by running the following command. The following command stores the index of the entry in an environment variable named `INDEX`:
```